src/styles/main.scss

```diff
@@ -0,0 +1,102 @@
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.box {
+  position: relative;
+
+  &__hour {
+    width: 10px;
+    height: 300px;
+    top: -300px;
+    background-color: red;
+    position: absolute;
+    transform-origin: bottom;
+    transform: rotate(0.01turn);
+    -webkit-animation-duration: 7s;
+    animation-duration: 7s;
+    -webkit-animation-name: move;
+    animation-name: move;
+  }
+
+  &__minute {
+    width: 10px;
+    height: 200px;
+    top: -200px;
+    background-color: #4b9aaa;
+    position: absolute;
+    transform-origin: bottom;
+    transform: rotate(0.01turn);
+    -webkit-animation-duration: 3s;
+    animation-duration: 3s;
+    -webkit-animation-name: move;
+    animation-name: move;
+  }
+
+  &__dot {
+    position: absolute;
+    background-color: #4b9aaa;
+    height: 30px;
+    width: 30px;
+    border-radius: 15px;
+    left: -10px;
+    top: -10px;
+  }
+}
+
+@-webkit-keyframes move {
```

Comment:
update animation. It should work as a real watch

https://user-images.githubusercontent.com/117587463/230969864-19f72bdc-d1a9-48fa-970b-d0757b071657.mov